src/scripts/components/example.es6

```diff
@@ -8,7 +8,7 @@ const Component = require('./component')
  * - DOM event listeners are in Backbone style
  *
  */
-module.exports = class Example extends Component {
+module.exports = class extends Component {
```

Comment:
Protože to jméno tady prostě nepoužíváš, ne? K té tříde se odnikud pomocí `Example` stejně nedostaneš, potřebuješ jít přes `require`. Je to to samý jako psát třeba
```
const foo = function whyIsThisNameHere() {
    //
}
```
→ Pokud to jméno nepotřebuješ k rekurzi, což v případě classes nepotřebuješ, je to tam zbytečné.
Možná to udělá nepatrně krásnější call stack, ale i to je díky source maps vedlejší. Jinak je to obecně prostě dead code.